ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -590,6 +580,10 @@ private enum UnityLayers {
                                  JaiFailliAttendre);
     // Timing4, 19.
     TimingManager.FixedUpdateAdd(TimingManager.TimingStage.Late, Late);
+    // Custom timing, 301.
+    planetarium_camera_adjuster_ =
+        gameObject.AddComponent<PlanetariumCameraAdjuster>();
+    planetarium_camera_adjuster_.adapter = this;
```

Comment:
Could this be set at construction?  (Thinking of perfect forwarding, but maybe KSP/Unity/C# are not smart enough.)

Reply:
I think it is discouraged to have constructors on `MonoBehaviours` (`AddComponent` constructs by reflection, so a parameterized constructor that takes the adapter is definitely out of the question).

Reply:
We could possibly do some Unity trickery to get it from the parent in `Awake` or something, to avoid the public mutability, but at that point I am not sure it would improve readability.